cmd/homi/setup/cmd.go

```diff
@@ -530,10 +535,10 @@ func RandStringRunes(n int) string {
 	return string(b)
 }
 
-func gen(ctx *cli.Context) error {
+func Gen(ctx *cli.Context) error {
 	genType := findGenType(ctx)
+	fmt.Println(genType)
```

Comment:
is this line intentional?

Reply:
Ohh, It was for debug... redundant code. Thanks